modules/javafx.controls/src/test/java/test/javafx/scene/control/ChoiceBoxTest.java

```diff
@@ -154,6 +155,26 @@ protected void startApp(Parent root) {
         assertNull(box.getSelectionModel());
     }
 
+    @Test public void testNullSelectionModelDoesNotThrowNPEOnValueChange() {
+        PrintStream defaultErrorStream = System.err;
+        ByteArrayOutputStream out = new ByteArrayOutputStream();
+        System.setErr(new PrintStream(out, true));
+
```

Comment:
hmm .. this looks a bit unusual. I suspect you want to somehow get around not receiving errors when property changes are involved - if so, the way we typically handle it is to install an uncaughtExceptionHandler to make them appear in the tests (see f.i. ListCellTest). Might have a reason I don't see, though - if so, which?

Reply:
Ahh, now I understand this. I saw that in one test class but was not sure what it does. I will have a look later on this.

Reply:
Done. Works like a charm, thanks for the tip. 😄